helper_test.go

```diff
@@ -2168,14 +2168,14 @@ func skipIfFreeOnly(t *testing.T) {
 }
 
 // skips a test if the test requires a beta feature
-func skipIfBeta(t *testing.T) {
+func skipUnlessBeta(t *testing.T) {
```

Comment:
The comment above should probably change to `// skips a test if beta feature is not enabled`

Reply:
I added a lot of detail in the comments about the usage of this skip